testing/platform/temporaltesting/temporal.go

```diff
@@ -0,0 +1,46 @@
+package temporaltesting
+
+import (
+	"context"
+	"io"
+
+	"github.com/stretchr/testify/require"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/testsuite"
+)
+
+var defaultNamespace = "temporaltesting-default-namespace"
```

Comment:
not configurable ?

Reply:
At the moment we don't have any applications using more than one namespace, so it didn't seem super important to add this feature.